pkg/resourceinterpreter/interpreter.go

```diff
@@ -12,16 +12,12 @@ import (
 	workv1alpha2 "github.com/karmada-io/karmada/pkg/apis/work/v1alpha2"
 	"github.com/karmada-io/karmada/pkg/resourceinterpreter/configurableinterpreter"
 	"github.com/karmada-io/karmada/pkg/resourceinterpreter/customizedinterpreter"
-	"github.com/karmada-io/karmada/pkg/resourceinterpreter/customizedinterpreter/webhook"
 	"github.com/karmada-io/karmada/pkg/resourceinterpreter/defaultinterpreter"
 	"github.com/karmada-io/karmada/pkg/util/fedinformer/genericmanager"
 )
 
-// ResourceInterpreter manages both default and customized webhooks to interpret custom resource structure.
-type ResourceInterpreter interface {
-	// Start starts running the component and will never stop running until the context is closed or an error occurs.
-	Start(ctx context.Context) (err error)
-
+// Interpreter describe the interpretation rules
```

Comment:
This comment seems not accurate, Interpreter can be used to query whether the specified action can be interpreted and performed the interpreting task.

/cc @RainbowMango